src/plopp/graphics/graphicalview.py

```diff
@@ -229,10 +240,11 @@ def render(self) -> None:
         all parent nodes and draw the figure.
         In addition, we call the home method to autoscale the axes and colormapper.
         """
-        self.draw_on_update = False
```

Comment:
Does this now trigger two redraws because drawing is no longer disabled in `update`?

Reply:
I think it does only one because of the `_autoscale_or_draw`?

Reply:
I'm not familiar enough with what happens in `super().render()`. But it looks to me like it would call `self.update`. And that calls `_autoscale_or_draw` which, because `self._autoscale == False`, draws. And then it calls `self.fit_to_data()` which draws a second time. Which one did I get wrong?

Reply:
I think you're right.

I changed the logic slightly, but to keep things simple, I'm still making multiple draw calls when `render` is called.
This seems to me as overkill to try and prevent that, as it's only called once when the figure is created, and only if there are multiple lines/artists.
It is not called at a high rate, so I think it's fine.

I can change it to only make a single call if you like, but the code will be more confusing I think.